connector_importer/models/recordset.py

```diff
@@ -141,9 +142,19 @@ def _set_serialized(self, fname, values, reset=False):
         """Update serialized data."""
         _values = {}
         if not reset:
-            _values = self[fname]
+            _values = getattr(self, fname) or {}
+            if _values:
+                _values = self._get_json_from_binary(_values)
+
         _values.update(values)
-        self[fname] = _values
+        json_report_data = json.dumps(_values)
+        _values = base64.b64encode(bytes(json_report_data, "utf-8"))
+        setattr(self, fname, _values)
+        # We need to invalidate the cache because the context dict
+        # bin_size=False triggers the _compute_datas(self) method
+        # which has the @api.depends_context('bin_size') decorator.
+        # Flush all pending computations and updates to the database.
+        self.env["ir.attachment"].invalidate_model()
```

Comment:
Can't we restrict this to `datas` field? 
I would say that we can do even better: get the attachment linked to this field and invalidate only that record.

Reply:
Indeed, nicely spotted. Many thanks